.github/workflows/build-arm64-wheels.yml

```diff
@@ -67,25 +71,12 @@ jobs:
         pip install setuptools_rust
         pip install twine
```

Comment:
Dont need to isntall twine. The new action will run in a container with twine in it